website/src/libs/api-v3/cron.js

```diff
@@ -181,7 +181,7 @@ export function cron (options = {}) {
     });
     task.completed = false;
 
-    if (completed || scheduleMisses > 0) {
+    if (!completed && scheduleMisses === 0) {
```

Comment:
I think the problem here is that you're marking the items as completed while instead they should be un-marked. See how it works in v2 https://github.com/HabitRPG/habitrpg/blob/develop/common/script/fns/cron.js#L276-L281